app/models/concerns/client_data_type.rb

```diff
@@ -1,14 +1,14 @@
-module ProposalDelegate
+module ClientDataType
```

Comment:
`ClientDataType` suggests it is an instantiate-able class. How about `ClientDataMixin` or `ClientDataConcern` or `ClientDataExtensions` ?